_builder_lib/docsitebuilder/helpers.rb

```diff
@@ -190,6 +202,7 @@ def page(args)
   <link href="#{args[:images_path]}respond.proxy.gif" id="respond-redirect" rel="respond-redirect">
   <script src="#{args[:javascripts_path]}respond.proxy.js" type="text/javascript"></script>
 <![endif]-->
+#{ANALYTICS_SHIM[args[:distro]]}
```

Comment:
Instead of doing the lookup here, within the `page()` method, do it down on line 574 where we call `page()` and hand in a hash of values. That will give you the opportunity to use the distro _keys_, instead of the pretty-print names, as the keys in your own hash.